### PR TITLE
Wrapper script for wkhtmltopdf. Wrapper shell script for upload.

### PR DESCRIPTION
**Pull request by mrchucho**
Depends on #1 

Ideal usage:

```bash
convert <URL> | upload <client>
```

### CODE REVIEW

File: bin/convert [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+INPUT=${1:-/dev/stdin}
+OUTPUT=${2:-/dev/stdout}
+
+JS_DELAY=1000
+
+/usr/local/bin/wkhtmltopdf \
+  --quiet \
+  --javascript-delay $JS_DELAY \
```
Comment (mrchucho) [inline bin/convert]: @natemcdowel I believe you indicated that this option is necessary. To what would you like it set?

Comment (mrchucho) [inline bin/convert]: Ah, missed this in the README. Updating to 10000.

File: bin/convert [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+INPUT=${1:-/dev/stdin}
+OUTPUT=${2:-/dev/stdout}
+
+JS_DELAY=10000
+
+/usr/local/bin/wkhtmltopdf \
+  --quiet \
+  --javascript-delay $JS_DELAY \
```
Comment (natemcdowel) [inline bin/convert]: This is handy..

**Comment by natemcdowel** [comment]
Looks good to me @mrchucho

**Pull request merged**